docs/topics/Permissions.md

```diff
@@ -39,18 +41,47 @@ Permissions in Discord are stored within a 53-bit integer and are calculated usi
 
 **\* These permissions require the owner account to use [two-factor authentication](#DOCS_OAUTH2/twofactor-authentication-requirement) when used on a guild that has server-wide 2FA enabled.**
 
+Note that these internal permission names may be referred to differently by the Discord client. For example, "Read Text Channels and See Voice Channels" refers to READ_MESSAGES; "Manage Permissions" refers to MANAGE_ROLES, and "Use Voice Activity" refers to USE_VAD.
```

Comment:
👍  maybe one day we'll be consistent

Reply:
READ_MESSAGES is technically now called `VIEW_CHANNEL`, so there is no consistency issue